.github/workflows/playwright.yml

```
@@ -0,0 +1,30 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [dev]
+  pull_request:
+    branches: [dev]
```

Whenever we merge some task branches into dev, we will trigger test.
I think we can trigger a new test once we create a pull request from dev to main.

```suggestion
    branches: [main]
```